### ISSUE TITLE
RH calculation ans Specific humidity calculation discrepancy in met2cf.ERA5.R

### ISSUE THREAD

**Issue by mccabete** [PecanProject/pecan#2897]
### Bug Description

@DongchenZ  Is the one who found this, I am just writing it up. The [relative humidity calculation](https://github.com/PecanProject/pecan/blob/develop/modules/data.atmosphere/R/met2cf.ERA5.R#L61) in met2cf.ERA5 has a * 100 term that put relative humidity in terms of percent, rather than proportion. 

According to the comments, this is copied from weathermetric's conversion function, [which labels that RH is in \%](https://github.com/geanders/weathermetrics/blob/master/R/moisture_conversions.R#L17). 

Our RH to specific humidity function is expecting [a proportion, not a \%](https://github.com/PecanProject/pecan/blob/ee38544e1c995f47073019bd31b3179e7e64991a/modules/data.atmosphere/R/metutils.R#L39). 

@DongchenZ's PR #2886 contains the fix, but I wanted to document the bug in case 1) I misinterpreted the use of `\%` in the descriptions, or 2) this affects anyone else

**Comment by mccabete** [comment]
@mdietze or @ankurdesai, and I interpreting the inputs to the [rh2qair()](https://github.com/PecanProject/pecan/blob/ee38544e1c995f47073019bd31b3179e7e64991a/modules/data.atmosphere/R/metutils.R#L45) function correctly?

**Comment by ankurdesai** [comment]
@mccabete Yes, the function expect RH has a proportion 0-1. e <- rh * es -> this estimates the vapor pressure as a proportion of saturation vapor pressure, that proportion is rh

**Comment by mccabete** [comment]
Thanks for confirming!
________________________________

**Comment by dlebauer** [comment]
Is it worth adding an argument to specify the units (proportion or percent) that RH is returned in.

Poked my head in and also noticed that rh2qair takes pressure in Pa, and qair2rh takes it in mb.
